cards.c

```diff
@@ -0,0 +1,136 @@
+/* Drew French */
+
+/* Draws a random card
+ * and prints the name
+ * and suit properly.
+ */
+
+ /* Didn't remember to use
```

Comment:
Because you forgot to do this, your code is incredibly slow and inefficient. You also should have included comments throughout.